File: Barberly/app/backend/Models/Booking.cs
using System.ComponentModel.DataAnnotations;

namespace backend.Models;

public class Booking
{
    [Key]
    public Guid bookingId { get; set; } = Guid.NewGuid();
    public Timeslot? timeslot { get; set; }
    public required string customerFirstName { get; set; }
    public required string customerLastName { get; set; }
    public required string customerEmail { get; set; }
    public string? customerPhoneNumber { get; set; }
}

File: Barberly/app/frontend/src/models/Booking.ts
import type { Timeslot } from "./Timeslot";

export interface Booking {
  bookingId: string;
  timeslot?: Timeslot;
  customerFirstName: string;
  customerLastName: string;
  customerEmail: string;
  customerPhoneNumber?: string;
}

export interface BookingDto {
  timeslotId: string;
  customerFirstName: string;
  customerLastName: string;
  customerEmail: string;
  customerPhoneNumber?: string;
}
